gen/test/java/testmodel/examples/ModelTestExample.java

```diff
@@ -0,0 +1,42 @@
+package testmodel.examples;
+
+import testmodel.examples.Model.*;
```

Comment:
is this Import necessary?

Reply:
Not everywhere, but for cases that use Embedded Enums(see for example: https://github.com/kokuwaio/micronaut-openapi-codegen/blob/328fe36f157e0b6c7807df4fc658c0eb122ccc85/gen/test/java/testmodel/examples/EnumerationModelTestExample.java). If you have an idea on how to identifiy such classes in the mustache we can put it only on those, but I didn't find one.